infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/LogReplicationEvent.java

```diff
@@ -57,7 +57,9 @@ public enum LogReplicationEventType {
      * @param type log replication event type
      */
     public LogReplicationEvent(LogReplicationEventType type) {
-        this(type, LogReplicationEventMetadata.empty());
+        this.type = type;
```

Comment:
let's remove the other unused constructor with explicit `eventId` as parameter to avoid confusion

Reply:
Are you referring to https://github.com/CorfuDB/CorfuDB/blob/log_replication_v2/infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/LogReplicationEvent.java#L72?  If so, yes I agree.  Let us remove it